testing/mod.ts

```diff
@@ -0,0 +1,175 @@
+/*!
+   Copyright 2018 Propel http://propel.site/.  All rights reserved.
```

Comment:
Isn't this copyright contradictory to MIT in root directory?

Reply:
Good catch. Fixed.